types/activex-libreoffice/activex-libreoffice-tests.ts

```diff
@@ -129,7 +129,7 @@
 
     function createStruct<K extends keyof LibreOffice.StructNameMap>(strTypeName: K): LibreOffice.StructNameMap[K] {
```

Comment:
@sandersn This would be better as:
```
function createStruct<K extends keyof LibreOffice.StructNameMap>(strTypeName: K): LibreOffice.StructNameMap[K] {
    const reflected = coreReflection.forName(strTypeName) as com.sun.star.reflection.XIdlClass<LibreOffice.StructNameMap[K]>;
    const aStruct: [LibreOffice.StructNameMap[K]] = [] as any;
    reflected.createObject(aStruct);
    return aStruct[0];
}
```

Reply:
Opened #34814 to do this.